lesson5/src/Zadanie25/Bankomat.java

```diff
@@ -0,0 +1,150 @@
+//Создать класс и объекты, описывающие Банкомат
+//
+//        Набор купюр, находящихся в банкомате должен задаваться 3мя свойствами: +
+//        * количеством купюр номиналом 20
+//        * -#-#-#-#                    50
+//        * -#-#-#-#                    100
+//
+//        Сделать методы для добавления денег в банкомат ++
+//
+//        Сделать, функцию снимающую деньги:
+//        На вход передается сумма денег, на выход - булевское значение(операция удалась или нет) ++
+//        При снятии денег, функция должна распечатывать, каким количством купюр, какого номинала выдается сумма ++
+//
+//        Создать конструктор с тремя параметрами - количеством купюр++
+
+package Zadanie25;
+
+public class Bankomat {
+
+    int amount_20;
+    int amount_50;
+    int amount_100;
+    int summa;
+
+    public int summa(){
+        summa = ((amount_20*20) + (amount_50*50) + (amount_100*100));
```

Comment:
you don't need addidtional bracets here. Just amount_20*20 + amount_50*50 + amount_100*100